Formula/pldiff.rb

```diff
@@ -0,0 +1,16 @@
+class Pldiff < Formula
+  desc "Compares plist files line by line"
+  homepage "https://github.com/scottrigby/pldiff"
+  url "https://github.com/scottrigby/pldiff/archive/1.0.4.tar.gz"
+  sha256 "0093027aee4ceb049902a9fcea28923477c87c2f88813e2370fb4c12195b9ccd"
+
+  depends_on "colordiff" => :recommended
+
+  def install
+    system "/bin/sh", "install.sh", prefix
```

Comment:
@woodruffw Gotcha, this makes sense.

I created a custom tap repo, and moved this formula there. I also made this change you suggested there as a separate commit https://github.com/scottrigby/homebrew-macos-configurator/commit/31d43aa55771b1d16903b264d26098b633f59eaa and gave props 😄 Thanks!